css/types/transform-function.json

```diff
@@ -192,6 +192,44 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "none": {
+            "__compat": {
+              "description": "`perspective(none)`",
+              "spec_url": "https://drafts.csswg.org/css-transforms-2/#funcdef-perspective",
+              "tags": [
+                "web-features:transforms3d"
+              ],
+              "support": {
+                "chrome": {
+                  "version_added": "97"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "93"
+                },
+                "firefox_android": "mirror",
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": "15.4"
```

Comment:
✅ https://github.com/WebKit/WebKit/commit/0e4a658cf874130d91a4c78dcd078c06feb31b84 in Safari [613.1.8](https://github.com/WebKit/WebKit/releases/tag/Safari-613.1.8) before 6.13.1.17 (15.4).